src/resources/Dissolutions/communityContributionCompany.ts

```diff
@@ -37,6 +37,16 @@ export const CommunityContributionCompanyDissolutionResource: DissolutionResourc
       ]
     }
   },
+  reviewAndConfirm: {
+    completingPartyStatement: {
+      certifyStatementHeader: null,
+      certifyStatements: [],
+      certifyClause: `Note: It is an offence to make a false or misleading statement in respect
+        of a material fact in a record submitted to the Corporate Registry for filing.
+        See section 427 of the Business Corporations Act.`,
+      entityDisplay: GetCorpFullDescription(CorpTypeCd.BC_CCC)
```

Comment:
Since entityDisplay always uses this method, why not move it into the HTML and remove it from here?

Reply:
... Unless you think it is much more clear to keep all this data together?

Reply:
This is the same structure we followed in IA resources

Reply:
Good answer :)